crates/yttrium/src/chain_abstraction/api/route.rs

```diff
@@ -47,8 +48,8 @@ pub struct RouteResponseAvailable {
 #[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
 #[serde(rename_all = "camelCase")]
 pub struct RouteResponseNotRequired {
-    #[serde(rename = "transactions")]
-    _flag: Vec<String>,
+    pub initial_transaction: Transaction,
```

Comment:
Not sure returning `initial_transaction` is necessary

Reply:
It is convenient to have - @llbartekll what's your take?

Reply:
yeah, I think it's good to have it

Reply:
Will it be estimated? I assume not. I think it's the same as the input

Reply:
In case the data is not ERC20 transfer we will simulate the transaction, estimate gas, and update the estimation for the initial tx.